ccd-definition/CaseEventToFields/CareSupervision/otherAllocationDecision.json

```
@@ -10,5 +10,17 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "otherAllocationDecision",
+    "CaseFieldID": "allocationProposal",
+    "PageFieldDisplayOrder": 1,
```

```suggestion
    "PageFieldDisplayOrder": 2,
```
Perhaps this should be a two just to make sure it won't clash (even though hidden)